src/pages/HomePage.module.css

```diff
@@ -0,0 +1,17 @@
+.div {
+  width: 50%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 500px;
+}
+
+.title {
+  font-size: 35px;
+}
```

Comment:
- [ ] OPTIONAL: The title on the home page is not properly aligned as it overlaps with the navbar. Kindly fix this when you get spare time. 
- [ ] OPTIONAL: The color contrast between different sections' backgrounds and font colors is a bit uncomfortable for me. For the sake of better UX, I recommend you change colors to bring better contrast.